Source/Utils/MPIInitHelpers.cpp

```diff
@@ -43,11 +52,12 @@ namespace utils
     }
 
     void
-    warpx_check_mpi_thread_level (std::pair< int, int > const mpi_thread_levels)
+    warpx_check_mpi_thread_level ()
     {
 #ifdef AMREX_USE_MPI
```

Comment:
```suggestion
int thread_required = 0;
int thread_provided = -1;
#ifdef AMREX_USE_MPI
```

Reply:
We need to define these variables before `#ifdef AMREX_USE_MPI` , otherwise we can't use them in `std::make_pair(thread_required, thread_provided);`

Reply:
Yes, but not here (above) :)